aws/resource_aws_storagegateway_smb_file_share_test.go

```diff
@@ -917,8 +912,8 @@ func testAccCheckAWSStorageGatewaySmbFileShareExists(resourceName string, smbFil
 	}
 }
 
-func testAccAWSStorageGateway_SmbFileShare_ActiveDirectoryBase(rName, domain string) string {
-	return composeConfig(testAccAWSStorageGatewayGatewayConfig_SmbActiveDirectorySettings(rName, domain), fmt.Sprintf(`
+func testAccAWSStorageGateway_SmbFileShare_ActiveDirectoryBase(rName, domainName string) string {
+	return composeConfig(testAccAWSStorageGatewayGatewayConfig_SmbActiveDirectorySettings(rName, domainName) + fmt.Sprintf(`
```

Comment:
`composeConfig()` already does the string concatenation, so the `+` isn't needed

Reply:
It was brought in from `main` today like that, I thought it looked a bit off. I'll make another commit and clean that up.